tests/pipelines/stable_diffusion/test_stable_diffusion.py

```diff
@@ -371,12 +373,6 @@ def test_stable_diffusion_prompt_embeds_with_plain_negative_prompt_list(self):
 
         assert np.abs(image_slice_1.flatten() - image_slice_2.flatten()).max() < 1e-4
 
-    def test_ip_adapter_single(self):
-        expected_pipe_slice = None
-        if torch_device == "cpu":
-            expected_pipe_slice = np.array([0.3203, 0.4555, 0.4711, 0.3505, 0.3973, 0.4650, 0.5137, 0.3392, 0.4045])
-        return super().test_ip_adapter_single(expected_pipe_slice=expected_pipe_slice)
```

Comment:
No point in keeping this test this way because we are reducing the model sizes dramatically so this won't contribute too much. LMK if that doesn't make sense.